eng/packages/http-client-csharp/generator/Azure.Generator/src/Utilities/ResourceDetection.cs

```diff
@@ -113,7 +145,7 @@ private static bool IsResourceModel(InputModelType inputModelType)
                     return true;
                 }
 
-                switch (property.SerializedName)
+                switch (property.Name)
```

Comment:
I think it should be serialized name, why we change it?

Reply:
fixed in https://github.com/Azure/azure-sdk-for-net/pull/48197/files#diff-fb6e63b8f76dfb7cde3dbed8e8d944507fb46a8f2d0b916b9ec2e815a5983aa1R116-R120